tests/lib/rules/jsx-uses-vars.js

```diff
@@ -115,6 +115,22 @@ ruleTester.run('no-unused-vars', rule, {
     }, {
       code: '\
         /*eslint jsx-uses-vars:1*/\
+        var App;\
+        var Hello;\
+        React.render(<App:Hello/>);',
```

Comment:
I had no idea this syntax was a thing (test added in e5e29cec / #219) - the [official docs](https://facebook.github.io/react/docs/jsx-in-depth.html#namespaced-components) don't mention it either.

@zertosh, could you verify this is correct? Also, can you link to where in the JSX spec this syntax is supported? If it's solely a custom addition then I don't think it's something we should try hard to support.